### PR TITLE
DOCS: Aligned/improved installation section package manager

### PR DESCRIPTION
**Pull request by cafferata**
The installation section within `README.md` and [Getting started](https://stackexchange.github.io/dnscontrol/getting-started#via-packages) has been aligned/improved in terms of 'package manager'.

**Before**
before

**After**
after

### CODE REVIEW

**Comment by cafferata** [comment]
I looked further at other projects how they build the README. Many projects link to the documentation from their README. What do you think of this idea? This prevents content diff as this merge request shows. If you think it's a good idea, I can of course provide a PR.

**Comment by tlimoncelli** [comment]
> I looked further at other projects how they build the README. Many projects link to the documentation from their README. What do you think of this idea? This prevents content diff as this merge request shows. If you think it's a good idea, I can of course provide a PR.

Not entirely sure what you mean.  Can you show me an example or a mock-up PR?

**Comment by cafferata** [comment]
See here two examples. I hope this better indicates which direction I want to go. If this is not the case, I will gladly prepare a draft PR.

> ## **Installation / Usage**
> 
> Download and install Composer by following the [official instructions](https://getcomposer.org/download/).
> 
> For usage, see [the documentation](https://getcomposer.org/doc/).

_https://github.com/composer/composer/#installation--usage_

> ## **Installation**
> 
> * [Install Symfony](https://symfony.com/doc/current/setup.html) with Composer (see [requirements details](https://symfony.com/doc/current/reference/requirements.html)).
> * Symfony follows the [semantic versioning](https://semver.org/) strictly, publishes "Long Term Support" (LTS) versions and has a [release process](https://symfony.com/doc/current/contributing/community/releases.html) that is predictable and business-friendly.

_https://github.com/symfony/symfony#installation_

**Comment by tlimoncelli** [comment]
Ok, now I see what you are proposing.  Yes, the README should link to install instructions.

**Comment by cafferata** [comment]
I've added an additional commit to clean up the README. See the before/after here.

**Before**
before


**After**
after

File: README.md [hunk 1]
```diff
@@ -139,60 +139,7 @@ See [Getting Started](https://stackexchange.github.io/dnscontrol/getting-started
...
-./runDocker.sh
-```
-
-FYI: Alternative Docker command:
-```bash
-docker run --rm -it --volume="$PWD:/srv/jekyll" --volume="$PWD/vendor/bundle:/usr/local/bundle" --env JEKYLL_ENV=production jekyll/jekyll:3.8 jekyll build -V
-# Open docs/_site/index.html in your web browser to see the results.
-# (Note: The preview isn't perfect. Links that use the site.github.url variable won't work.
-```
+Install DNSControl by following the [official instructions](https://stackexchange.github.io/dnscontrol/getting-started#1-install-the-software).
```
Comment (tlimoncelli) [inline README.md:142]: Suggested change:

DNSControl can be installed via packages for macOS, Linux and Windows, or from source code. See the [official instructions](https://stackexchange.github.io/dnscontrol/getting-started#1-install-the-software).

File: docs/getting-started.md [hunk 2]
```diff
@@ -31,9 +31,21 @@ If these don't work, more info is in [#805](https://github.com/StackExchange/dns
 
 Get prebuilt binaries from [GitHub releases](https://github.com/StackExchange/dnscontrol/releases/latest)
 
-Alternatively, on Mac you can install it using homebrew:
+## Through a package manager
```
Comment (tlimoncelli) [inline docs/getting-started.md:34]: Suggestion: Give the package instructions first, then the source code instructions.  That way we encourage the use of the packages.

File: docs/getting-started.md [hunk 3]
```diff
@@ -31,9 +31,21 @@ If these don't work, more info is in [#805](https://github.com/StackExchange/dns
...
 
-`brew install dnscontrol`
+Alternatively, on macOS (or Linux) you can install it using [Homebrew](https://brew.sh) or [MacPorts](https://www.macports.org):
```
Comment (tlimoncelli) [inline docs/getting-started.md:36]: Binaries are also available for Linux (binary, tar, RPM, DEB), FreeBSD (tar), Windows (exec, ZIP) for 32-bit, 64-bit, and ARM at https://github.com/StackExchange/dnscontrol/releases

Could you somehow work that into the doc?

Comment (cafferata) [reply docs/getting-started.md:36]: Yep, included in the last commit.

**Comment by cafferata** [comment]
See here the processed feedback.

**Before** 
before

**After**
after

**Comment by tlimoncelli** [comment]
Oh this is so much better! Thank you (again) for these updates!

**Pull request merged**